app/workers/zync_worker.rb

```diff
@@ -13,6 +13,15 @@ class ZyncWorker
 
   sidekiq_options queue: :zync
 
+  # It is used as the delay, in seconds.
+  # The default variable is the same as sidekiq's default, as you can see in https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry
+  # the default would last up 20 days and some hours, and we need it to last less than 7 days for the event to still exist,
+  # so it goes 3 times faster to last maximum 6 days and some hours (1/3 = 7/21)
+  sidekiq_retry_in do |count, _exception|
```

Comment:
Just a reminder that this has no effect over the `UnprocessableEntityError` errors now that we are not really raising them anymore. Those are retried immediately after the batch of dependencies completes.

Reply:
That's a good thing, but what this PR means to fix is:
We have a spike in production that the retries are failing because the event doesn't exist anymore... that happens because it retries up to 25 times in a bit over 20 days.
The retry can be initially for any reason, even that Zync is down 😄 
So what this does is just splitting those retries to be 3 times faster so the event will still exist (and 1 week should be enough 😑 )